sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/CHANGELOG.md

```diff
@@ -1,16 +1,8 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
 
-## 1.0.0-beta.1 (2024-11-22)
```

Comment:
I think this change is incorrect
Please revert this and add some briefs about your changes in this PR in `1.0.0-beta.2`

Reply:
done